### PR TITLE
Add Deriving 0.1.1.

### PR DESCRIPTION
**Pull request by arthuraa**

### CODE REVIEW

File: released/packages/coq-deriving/coq-deriving.0.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+homepage: "https://github.com/arthuraa/deriving"
+bug-reports: "https://github.com/arthuraa/deriving/issues"
+dev-repo: "git+https://github.com/arthuraa/deriving.git"
+license: "MIT"
+
+build: [ make "-j" "%{jobs}%" "test" {with-test} ]
+install: [ make "install" ]
+depends: [
+  "coq" { (>= "8.11" & < "8.18~") | (= "dev") }
+  "coq-mathcomp-ssreflect" {>= "1.11" | (= "dev")}
```
Comment (palmskog) [inline released/packages/coq-deriving/coq-deriving.0.1.1/opam:15]:
```suggestion
  "coq-mathcomp-ssreflect" {>= "1.11"}
```
The `| (= "dev")` is redundant.

File: released/packages/coq-deriving/coq-deriving.0.1.1/opam [hunk 2]
```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+name: "coq-deriving"
+version: "0.1.1"
```
Comment (palmskog) [inline released/packages/coq-deriving/coq-deriving.0.1.1/opam:3]:
```suggestion
```
These are already given by the file path.

Comment (arthuraa) [reply released/packages/coq-deriving/coq-deriving.0.1.1/opam]: Done.

File: released/packages/coq-deriving/coq-deriving.0.1.1/opam (hunk 1 above)
Comment (arthuraa) [reply released/packages/coq-deriving/coq-deriving.0.1.1/opam]: Done.

**Pull request merged**